lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/RemoteCentralRepositorySearcher.java

```diff
@@ -265,24 +270,41 @@ private OkHttpAetherClient newHttpClient() {
 				proxyPort = Integer.getInteger(System.getProperty("https.proxyPort"));
 			}
 		}
-		if (proxyHost != null && proxyPort != null) {
-			AetherClientProxy clientProxy = new AetherClientProxy();
-			clientProxy.setHost(proxyHost);
-			clientProxy.setPort(proxyPort);
-			clientProxy.setAuthentication(authentication);
-			config.setProxy(clientProxy);
-		}
-
-		if (config.getProxy() == null) {
-			config.setAuthentication(authentication);
-		}
-		
+//		if (proxyHost != null && proxyPort != null) {
+//			HttpProxy clientProxy = new AetherClientProxy();
+//			clientProxy.setHost(proxyHost);
+//			clientProxy.setPort(proxyPort);
+//			clientProxy.setAuthentication(authentication);
+//			
+//			config.setProxy(clientProxy);
+//		}
+
+//		if (config.getProxy() == null) {
+//			config.setAuthentication(authentication);
+//		}
+//		
 		// ex: LemMinX/0.27.1-SNAPSHOT (Windows 11 10.0)
 		String userAgent = "LemMinX/" + Platform.getVersion().getVersionNumber() + " (" + Platform.getOS().getName()
 				+ " " + Platform.getOS().getVersion() + ")";
-		config.setUserAgent(userAgent);
-		config.setHeaders(Collections.emptyMap());
-		return new OkHttpAetherClient(config);
+		
+		RequestConfig requestConfig = RequestConfig.custom()
+				.setConnectionRequestTimeout(ConfigurationProperties.DEFAULT_REQUEST_TIMEOUT)
+				.setConnectTimeout(3*ConfigurationProperties.DEFAULT_CONNECT_TIMEOUT)
+				.build();
+
+		HttpClientBuilder builder = HttpClientBuilder.create() //
+	            .useSystemProperties() //
+	            .disableConnectionState() //
+	            .setUserAgent(userAgent) //
+	            .setDefaultHeaders(Collections.emptySet()) //
+	            .setDefaultRequestConfig(requestConfig); //
+
+		if (proxyHost != null && proxyPort != null) {
+			builder = builder.setProxy(
+					new HttpHost(proxyHost, proxyPort, null));
+		}
```

Comment:
If using standard java.net.HttpClient, I think support for proxies based on system properties is built-in and no extra configuraiton is necessary.

Reply:
Still should read and provide the host/port requisites to create a proxy.